### DISCUSSION TITLE
can't open source in xcode

### DISCUSSION THREAD

**Discussion by yandm1tr** [OutlineFoundation/outline-sdk#199] [Q&A]
I'm trying to open the source outline

but when loading it into xcode the following situation comes out, who knows what it can be connected with?
<img width="271" alt="Снимок экрана 2024-03-15 в 12 39 45" src="https://github.com/Jigsaw-Code/outline-sdk/assets/163401977/8072d21d-8ce1-49e6-911c-393eb5d20ab3">

my course of action:
- downloaded the entire repository from github,
- opened the project folder with xcode
- got the same situation

**Comment by fortuna** [comment]
Please file this in https://github.com/Jigsaw-Code/outline-apps/issues/new/choose instead. This discussion group is for the Outline SDK only, not the Outline app.
